src/particle/particle/particle.py

```diff
@@ -1124,9 +1121,6 @@ def finditer(
         See also ``findall``, which returns the same thing, but as a list.
         """
 
-        # Note that particle can be called by position to keep compatibility with Python 2, but that behavior should
-        # not be used and will be removed when support for Python 2.7 is dropped.
-
```

Comment:
I've implemented this now, as promised.

Reply:
To what are you refering to?

Reply:
`finditer(function, True)` is no longer allowed, now it must be `finditer(function, particle=True)`. Though I missed `findall` just below. Will fix.